sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -557,61 +557,62 @@ static void _send_multicast(gnrc_pktsnip_t *pkt, bool prep_hdr,
     }
 
 
```

Comment:
This line is too much [according to Vera++](https://travis-ci.org/github/RIOT-OS/RIOT/builds/660563543#L227). I guess it does not hurt much to remove it in this PR.